<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>481</width>
      <height>901</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>16777211</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>361</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bradley Hand</family>
        <pointsize>24</pointsize>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>How are you feeling right now?</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnHappy">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>190</y>
        <width>131</width>
        <height>141</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../icons/happiness.png</normaloff>../../../icons/happiness.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>126</width>
        <height>126</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnNeutral">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>370</y>
        <width>131</width>
        <height>141</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../Downloads/icons/neutral.png</normaloff>../../../../../Downloads/icons/neutral.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>126</width>
        <height>126</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSad">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>370</y>
        <width>131</width>
        <height>131</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../Downloads/icons/sad(1).png</normaloff>../../../../../Downloads/icons/sad(1).png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>126</width>
        <height>126</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRelaxed">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>190</y>
        <width>131</width>
        <height>141</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../Downloads/icons/relax.png</normaloff>../../../../../Downloads/icons/relax.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>126</width>
        <height>126</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAnxious">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>540</y>
        <width>141</width>
        <height>141</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../Downloads/icons/depressed.png</normaloff>../../../../../Downloads/icons/depressed.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>126</width>
        <height>126</height>
       </size>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="btnStressed">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>540</y>
        <width>131</width>
        <height>141</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../../../../../Downloads/icons/angry.png</normaloff>../../../../../Downloads/icons/angry.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>126</width>
        <height>126</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bradley Hand</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Happy</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>340</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bradley Hand</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Sad</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>510</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bradley Hand</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Stressed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>170</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bradley Hand</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Relaxed</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>340</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bradley Hand</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Neutral</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>510</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bradley Hand</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Anxious</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Bradley Hand</family>
        <pointsize>24</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Improve your mental health</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>420</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>311</width>
        <height>141</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="11.qrc"/>
 </resources>
 <connections/>
</ui>
